lib/genome/importers/drug_bank/new_drug_bank.rb

```diff
@@ -0,0 +1,127 @@
+module Genome; module Importers; module DrugBank;
+  class NewDrugBank
+    attr_reader :file_path, :source
+    def initialize(file_path)
+      @file_path = file_path
+    end
+
+    def import
+      remove_existing_source
+      create_source
+      import_claims
+    end
+
+    def get_version
+      File.open('lib/genome/updaters/data/version').readlines.each do |line|
+        source, version = line.split("\t")
+        if source == 'DrugBank'
+          return version.strip
+        end
+      end
+      return nil
+    end
+
+    private
+    def remove_existing_source
+      Utils::Database.delete_source('DrugBank')
+    end
+
+    def create_source
+      @source = DataModel::Source.where(
+        base_url: 'http://www.drugbank.ca',
+        site_url: 'http://www.drugbank.ca/',
+        citation: "DrugBank 4.0: shedding new light on drug metabolism. Law V, Knox C, Djoumbou Y, Jewison T, Guo AC, Liu Y, Maciejewski A, Arndt D, Wilson M, Neveu V, Tang A, Gabriel G, Ly C, Adamjee S, Dame ZT, Han B, Zhou Y, Wishart DS.Nucleic Acids Res. 2014 Jan 1;42(1):D1091-7. PubMed ID: 24203711",
+        source_db_version: get_version,
+        source_type_id: DataModel::SourceType.INTERACTION,
+        source_db_name: 'DrugBank',
+        full_name: 'DrugBank - Open Data Drug & Drug Target Database'
+        #source_trust_level_id: DataModel::SourceTrustLevel.EXPERT_CURATED
+      ).first_or_create
+    end
+
+    def import_claims
+      blank_filter = ->(x) { x.blank? || x == "''" || x == '""' || x.downcase.tr('/','') == 'na'}
```

Comment:
@susannasiebert Do lines 43-45 need to be here? They look like an artifact from the old tsv grouper.